src/shared/locales/all.js

```diff
@@ -132,6 +134,12 @@ const resources = {
       ...appLocalePl
     }
   },
+  'nb': {
```

Comment:
Please adjust the order of 'nb' to before 'pl' .

Reply:
:)